content/events/2025-sigcomm-ebpf-workshop/index.md

```diff
@@ -0,0 +1,12 @@
+---
+date: '2025-09-07T10:00:00.000Z'
+region: EMEA
+place: Coimbra, Portugal
+title: 'SIGCOMM eBPF Workshop 2025'
+description: "ACM SIGCOMM is the flagship academic conference in the field of networking. The 3rd eBPF workshop took place at this year's ACM SIGCOMM conference."
```

Comment:
Feels strange to use past tense for something due to take place next September, but if that's intended then OK.

Reply:
Yeah, didn't really think it through. Updated.